api/gdpr.py

```diff
@@ -0,0 +1,165 @@
+                applications.append(
+                    {
+                        "contact_person": [
+                            app.contact_person.first_name,
+                            app.contact_person.last_name,
+                            app.contact_person.email,
+                            app.contact_person.phone_number,
+                        ]
+                    }
+                )
+
+            if app.organisation:
+                applications.append(
+                    {
+                        "organisation": [
+                            app.organisation.name,
+                            app.organisation.identifier,
+                            app.organisation.email,
+                            app.organisation.core_business,
+                            app.organisation.core_business_fi,
+                            app.organisation.core_business_en,
+                            app.organisation.core_business_sv,
+                        ]
+                    }
+                )
+            if app.organisation and app.organisation.address:
+                applications.append(
+                    {
+                        "organisation_address": [
+                            app.organisation.address.post_code,
+                            app.organisation.address.street_address,
+                            app.organisation.address.street_address_fi,
+                            app.organisation.address.street_address_en,
+                            app.organisation.address.street_address_sv,
+                            app.organisation.address.city,
+                            app.organisation.address.city_fi,
+                            app.organisation.address.city_en,
+                            app.organisation.address.city_sv,
+                        ]
+                    }
+                )
+            if app.billing_address:
+                applications.append(
+                    {
+                        "billing_address": [
+                            app.billing_address.post_code,
+                            app.billing_address.street_address,
+                            app.billing_address.street_address_fi,
+                            app.billing_address.street_address_en,
+                            app.billing_address.street_address_sv,
+                            app.billing_address.city,
+                            app.billing_address.city_fi,
+                            app.billing_address.city_en,
+                            app.billing_address.city_sv,
+                        ]
+                    }
+                )
+
+        return applications
+
+    @property
+    def email(self):
+        return self.user.email
+
+
+class TilavarauspalveluGDPRAPIView(GDPRAPIView):
+    def get_object(self) -> User:
+        user = get_object_or_404(User, uuid=self.kwargs["uuid"])
+        profile = ProfileUser(user)
+        # Permissio check???
```

Comment:
Not sure if this comment is necessary here. What is it's function?

Reply:
it is dirt I'll clean it up